core/test/bundles/execWithStaticBundle/main.cpp

```diff
@@ -0,0 +1,51 @@
+#include <cstdio>
+#include <iostream>
+#include <string>
+#include <cassert>
+
+#include "usFrameworkFactory.h"
+#include "usFramework.h"
+#include "usBundleContext.h"
+#include "usBundleImport.h"
+
+#include "usTestingMacros.h"
+#include "usTestingConfig.h"
+#include "usTestBundleService.h"
+
+using namespace us;
+
+/**
+ * Test point to ensure the statically linked bundle is installed properly.
+ */
+int main(int ac, char **av)
+{
+  auto framework = FrameworkFactory().NewFramework();
+  framework->Start();
+  BundleContext* frameworkCtx = framework->GetBundleContext();
+  auto bundle = frameworkCtx->InstallBundle(BIN_PATH + DIR_SEP + "usTestExecutableWithStaticBundle" + EXE_EXT + "/TestStaticBundle");
```

Comment:
We still need the bundle name to install. The change you are referring to is only in the EmbeddedBundles.